doc/doxygen/CMakeLists.txt

```diff
@@ -300,6 +300,10 @@ ADD_CUSTOM_TARGET(doxygen ALL
   )
 ADD_DEPENDENCIES(documentation doxygen)
 
+# Set the canonical link for the doxygen webpages
+ADD_CUSTOM_COMMAND(TARGET doxygen POST_BUILD
+    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/../../contrib/utilities/set_canonical_doxygen.py)
```

Comment:
Yes, please relocate this script to `./doc/doxygen/scripts/`. This also avoids the `..` in the path.